bucket/filezilla.json

```
@@ -15,10 +15,15 @@
     },
     "extract_dir": "FileZilla-3.47.1",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
-        "}"
+        "@\"",
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>",
+        "<FileZilla3>",
+        "  <Settings>",
+        "    <Setting name=\"Config Location\">$dir\\config</Setting>",
```

I do not understand your mind flow at all ... You say it is not needed to use fullpath, but you still change it...

```suggestion
        "    <Setting name=\"Config Location\">config</Setting>",
```

---

Both are working. I was thinking you were preferring the absolute path. Sorry for the confusion